src/annotator/integrations/test/image-text-layer-test.js

```diff
@@ -150,37 +168,68 @@ describe('ImageTextLayer', () => {
       imageText
     );
 
-    assert.equal(textLayer.container.textContent, 'first line second line ');
-    const wordSpans = getWordBoxes(textLayer);
-    assert.equal(wordSpans.length, imageText.split(/\s+/).length);
+    assert.equal(textLayer.container.textContent, 'first line second line');
+    const wordBoxes = getWordBoxes(textLayer);
+    assert.equal(wordBoxes.length, imageText.split(/\s+/).length);
     assert.deepEqual(
-      wordSpans.map(ws => ws.textContent),
-      ['first', 'line', 'second', 'line']
+      wordBoxes.map(ws => ws.textContent),
+      ['first ', 'line ', 'second ', 'line']
     );
 
     const imageBox = image.getBoundingClientRect();
-    const wordBoxPositions = wordSpans.map(span => {
-      const wordBox = span.getBoundingClientRect();
+    const wordBoxPositions = wordBoxes.map(wordEl => {
+      const rect = untransformedBoundingRect(wordEl);
       return [
-        wordBox.left - imageBox.left,
-        wordBox.top - imageBox.top,
-        wordBox.width,
-        wordBox.height,
-      ].map(coord => Math.floor(coord));
+        rect.left - imageBox.left,
+        rect.top - imageBox.top,
+        rect.width,
+        rect.height,
+      ].map(coord => Math.round(coord));
```

Comment:
I used rounding here because I'm generating a list of numbers that I want to compare, whereas `assert.approximately` only works with individual values.